### ISSUE TITLE
Define an MVP (clone from abbra)

### ISSUE THREAD

**Issue by johnkeates** [d3vi1/freeipa-macosx-support#1]
I posted this at abbra's repo: 

> I think it would help everyone involved if we made a list of things that are needed to get a basic version working. For example:
> 
> - Being able to generate a profile
> - Said profile can configure a machine to do auth from the login screen and ssh sessions
> - Said profile can be generated from the cli

I'm not sure if this helps here as well, but it might if we are to contribute to your fork first.

**Comment by d3vi1** [comment]
Terminology wise,  let's call the profile a template as 'profile' is already taken by the AppleConfigurator.
Said profile is generated automatically once the macOS plugin is enabled in FreeIPA.

**Comment by johnkeates** [comment]
Yes, let's call it that. Basically, the goal would be IPA producing a file, which would basically do a 1-click configuration of a macOS client.

**Comment by d3vi1** [comment]
I've created an initial Wiki page on the project with the current list of identified tasks and what directions I've identified for each of them. There are a few where I will need some help (IPA specific). Drop by.
